src/com/pentalog/pentastagiu/homework2/Shop/Product.java

```diff
@@ -0,0 +1,37 @@
+package com.pentalog.pentastagiu.homework2.Shop;
+
+public class Product {
+    private int id;
+    private String name;
+    private double price;
+
+    public Product(int id, String name, double price) {
+        this.id = id;
```

Comment:
pentru id ar fi mers un mecanism de generare, teoretic id-ul este un identifier pentru un produs si nu trebuie setat; o varainta pentru generarea id-ului ar fi fost setarea lui aici in constructor cu valoarea unui field static care iti numara cate obiecte au fost create din clasa respetiva